src/main/java/entities/MonthObject.java

```diff
@@ -5,6 +5,7 @@
  * they can be created by MonthObjectFactory classes. It contains the setName and getName methods that all MonthObject
  * classes should have.
  */
+@SuppressWarnings("unused") // These methods are actually used, but just on the concrete class
```

Comment:
What is the reasoning for having this here?

Reply:
It's to disregard an IntelliJ warning explicitly. Otherwise, IntelliJ gives a warning because we don't use the methods getName and setName anywhere with a MonthObject directly, but we do use them with the concrete classes, so it's not actually an unused method.

Reply:
Adding the decorator to the class makes IntelliJ not show that warning for the methods inside the class, and then I added a small comment explaining why it's ok to disregard this warning

Reply:
Makes sense, thank you!